tip-distributor/src/lib.rs

```diff
@@ -513,12 +521,17 @@ pub async fn sign_and_send_transactions_with_retries(
                     }
                     Err(e) => {
                         error!("error sending transaction {sig:?} error: {e:?}");
-                        Err(e)
+                        if let ErrorKind::TransactionError(AlreadyProcessed) = e.kind {
+                            Ok(())
+                        } else {
+                            Err(e)
```

Comment:
maybe move line 523 to be inside the block so we aren't spammed with already processed msgs